README.md

```diff
@@ -2,7 +2,8 @@
 1. [Download your Twitter archive](https://twitter.com/settings/download_your_data) (Settings > Your account > Download an archive of your data).
 2. Unzip to a folder.
 3. Right-click this link --> [parser.py](https://raw.githubusercontent.com/timhutton/twitter-archive-parser/main/parser.py) <-- and select "Save Link as", and save into the folder where you extracted the archive. (Or use wget or curl on that link. Or clone the git repo.)
-4. Run parser.py with [Python 3](https://realpython.com/installing-python/). e.g. `python parser.py` from a command prompt opened in that folder.
+4. Change into the unzipped folder where you just saved parser.py.
```

Comment:
Needs to include 'Open a command prompt'.

Maybe something like 'Open a command prompt into that same folder.'

And then remove 'from a command prompt in that folder from the last step.